runtime-display/glew/autobuild/defines

```diff
@@ -11,3 +11,4 @@ PKGBREAK="avogadro<=1.2.0-6 avogadrolibs<=1.92.1-1 blastem<=0.6.2 blender<=2.81a
           opensubdiv<=3.3.3-4 pcsx2<=1.4.0-5 ppsspp<=1.9.4 projectm<=2.1.0-4 \
           rlvm<=0.14.67-2 sfml<=2.5.0 slop<=7.4-3 supertux<=0.6.1 supertuxkart<=1.1 \
           vice<=3.3-1 warzone2100<=3.2.3-4 yabause<=0.9.15-1"
+NOLTO__LOONGARCH64=1
```

Comment:
Please move this above `PKGBREAK` and add a `# FIXME:` comment explaining why.

Reply:
Done.